tests/system/migrations/test_0100_init_reldb.py

```diff
@@ -0,0 +1,527 @@
+                        "username": ADMIN_USERNAME,
+                        "password": self.auth.hash(ADMIN_PASSWORD),
+                    },
+                }
+            )
+            # Login and save copy of auth data for all following tests
+            self.client.login(ADMIN_USERNAME, ADMIN_PASSWORD, 1)
+            BaseMigrationTestCase.auth_data = deepcopy(self.client.auth_data)
+
+        self.setup_data()
+        self.apply_test_relational_schema()
+
+    def request(
+        self,
+        command: str,
+        lang: str | None = None,
+    ) -> Response:
+        headers = get_internal_auth_header(DEV_PASSWORD)
+        if lang:
+            headers["Accept-Language"] = lang
+        response = self.client.post(
+            MIGRATIONS_URL, json={"cmd": command, "verbose": True}, headers=headers
+        )
+        if response.status_code == 200:
+            results = response.json.get("results", {})
+            assert results == {}
+        return response
+
+    def setup_data(self) -> None:
+        raw_data: dict[str, any]
+        json_blob: str
+
+        # 2) reading json data from file
+        with open(EXAMPLE_DATA_PATH) as file:
+            raw_data = json.loads(file.read())
+
+        # 2.1) fill data dictionary without meta_ fields and _migration_index
+        for collection, models in raw_data.items():
+            if collection == "_migration_index":
+                continue
+            for model_id, model in models.items():
+                data[f"{collection}/{model_id}"] = {
+                    f: v for f, v in model.items() if not f.startswith("meta_")
+                }
+
+        # 3) Open os_connection_pool
+        os_conn_pool.open()
+
+        # 4) Write models into db table models
+        with os_conn_pool.connection() as conn:
+            with conn.cursor() as curs:
+                # 4.D1) clears models table
+                curs.execute("TRUNCATE TABLE models;")
+
+                # 4.1) Actual writing of models into table
+                for fqid, model in data.items():
+                    json_blob = json.dumps(model)
+                    curs.execute(
+                        "INSERT INTO models VALUES (%s, %s, false, now());",
+                        [fqid, json_blob],
+                    )
+
+                # 4.2) Write Migration Index like Docker setup would expect
+                curs.execute(
+                    f"INSERT INTO positions (timestamp, user_id, migration_index) VALUES ('2020-05-20', 1, {LAST_NON_REL_MIGRATION + 1})"
+                )
+
+    def check_data(self) -> None:
+        # 6) TEST CASES
+        with os_conn_pool.connection() as conn:
+            with conn.cursor() as cur:
+                # 6.1) 1:1 relation
+                result = cur.execute(
+                    "SELECT theme_id FROM organization_t WHERE id=1;"
+                ).fetchone()
+                assert result is not None, "1:1 relation in organization not filled."
```

Comment:
This part and below:
Sometimes you're writing `result = cur.execute("...").fetchone()` and then `assert result = ...`, and sometimes you're writing `cur.execute("...")` and then `cur.fetchone()`.

Two questions:
1. Why not more consistent?
2. Why not make a helper function that hides the repeating parts of this pattern for the sake of readability?